src/main/java/com/github/dockerjava/core/AbstractDockerCmdExecFactory.java

```diff
@@ -138,7 +138,7 @@
 
 public abstract class AbstractDockerCmdExecFactory implements DockerCmdExecFactory {
 
-    private DockerClientConfig dockerClientConfig;
+    protected DockerClientConfig dockerClientConfig;
```

Comment:
To fix the compilation issue.

Reply:
What issue?

Reply:
dockerClientConfig is not visible to the subclass com.github.dockerjava.netty.NettyDockerCmdExecFactory line 322
```
'dockerClientConfig' has private access in AbstractDockerCmdExecFactory
```
which causes compilation failure.

Or, in NettyDockerCmdExecFactory line 322, use getDockerClientConfig(), such as:
```
inal Object dockerAPIEndpoint = getDockerClientConfig().getDockerHost();
```